app/experimenter/legacy-ui/templates/experiments/nimbus-ui.html

```diff
@@ -0,0 +1,30 @@
+{% load static %}
+
+<!DOCTYPE html>
```

Comment:
Would it be possible for this to live somewhere outside of `legacy-ui` since we want to delete this directory down the line? I'm not sure of the best placement, I mean even `app/experimenter/nimbus-ui.html` could make sense as it'd be adjacent to `nimbus-ui/`.

Reply:
Oh yeah we can put it in `nimbus-ui/templates` and just add it here:

https://github.com/mozilla/experimenter/blob/main/app/experimenter/settings.py#L94-L97